easybuild/easyconfigs/a/artic-ncov2019/artic-ncov2019-2021.06.24-foss-2020b.eb

```diff
@@ -0,0 +1,124 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+# Updated to latest version 24. Jun 2021 and bumped to foss-2020b
+# to use it with pangolin
+# J. Sassmannshausen (GSTT/NHS UK)
+
+easyblock = 'PythonBundle'
+
+name = 'artic-ncov2019'
+local_commit = 'b3f2dda5e6d95bc1c5d95a04d4ef37d304479477'
+version = '2021.06.24'
+
+homepage = "https://github.com/artic-network/artic-ncov2019"
+description = """Initial implementation of an ARTIC bioinformatics platform 
+for nanopore sequencing of nCoV2019 novel coronavirus."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Biopython', '1.78'),
+    ('Pysam', '0.16.0.1'),
+    ('BWA', '0.7.17'),
+    ('minimap2', '2.18'),
+    ('SAMtools', '1.11'),
+    ('MUSCLE', '3.8.31'),
+    ('ETE', '3.1.2'),
+    ('MAFFT', '7.475', '-with-extensions'),
+    ('IQ-TREE', '2.1.2'),
+    ('snakemake', '6.1.0'),
+    ('Longshot', '0.4.3'),
+    ('medaka', '1.4.3'),
+    ('python-parasail', '1.2.4'),
+    ('PhyML', '3.3.20200621'),
+    ('nodejs', '12.19.0'),
+    ('goalign', '0.3.2', '', True),
+    ('gotree', '0.4.0', '', True),
+    ('rampart', '1.2.0'),
+    ('libdeflate', '1.7'),
+    ('nanopolish', '0.13.3'),
+    ('seqtk', '1.3'),
+    ('BCFtools', '1.11'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('zipp', '3.5.0', {
+        'checksums': ['f5812b1e007e48cff63449a5e9f4e7ebea716b4111f9c4f9a645f91d579bf0c4'],
+    }),
+    ('pluggy', '1.0.0', {
+        'checksums': ['4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159'],
+    }),
+    ('py', '1.10.0', {
+        'checksums': ['21b81bda15b66ef5e1a777a21c4dcd9c20ad3efd0b3f817e7a809035269e1bd3'],
+    }),
+    ('wcwidth', '0.2.5', {
+        'checksums': ['c4d647b99872929fdb7bdcaa4fbe7f01413ed3d98077df798530e5b04f116c83'],
+    }),
+    ('attrs', '21.2.0', {
+        'modulename': 'attr',
+        'checksums': ['ef6aaac3ca6cd92904cdd0d83f629a15f18053ec84e6432106f7a4d04ae4f5fb'],
+    }),
+    ('importlib-metadata', '4.8.1', {
+        'modulename': 'importlib_metadata',
+        'source_tmpl': 'importlib_metadata-%(version)s.tar.gz',
+        'checksums': ['f284b3e11256ad1e5d03ab86bb2ccd6f5339688ff17a4d797a0fe7df326f23b1'],
+    }),
+    ('more-itertools', '8.9.0', {
+        'modulename': 'more_itertools',
+        'checksums': ['8c746e0d09871661520da4f1241ba6b908dc903839733c8203b552cffaf173bd'],
+    }),
+    ('args', '0.1.0', {
+        'checksums': ['a785b8d837625e9b61c39108532d95b85274acd679693b71ebb5156848fcf814'],
+    }),
+    ('pytest', '6.2.5', {
+        'checksums': ['131b36680866a76e6781d13f101efb86cf674ebb9762eb70d3082b6f29889e89'],
+    }),
+    ('tqdm', '4.62.2', {
+        'checksums': ['a4d6d112e507ef98513ac119ead1159d286deab17dffedd96921412c2d236ff5'],
+    }),
+    ('fieldbioinformatics', '1.2.1', {
+        'modulename': False,
```

Comment:
```suggestion
        'modulename': 'artic',
```

Reply:
I done the changes. Before it gets merged, I want to have a chat with a user to make sure we are not removing things which they actually use. I am thinking of modules like for example `importlib-metadata` and `more-itertools` .

Reply:
Some more on that: I am not quite sure if removing these modules is a good way forward, from a practical point of view. 
For example, removing `importlib-metadata` results:
`importlib-metadata            2.0.0`
whereas leaving it in means we got a newer version:
`importlib-metadata            4.8.1`
From a puristic point of view you are right, no questions here. From a practical point of view, I need to speak to one of my users (as above). 
What is your take here?

Reply:
Unless that stuff is not required I don't see a reason to include them. In pretty much all ECs we only include the actually required stuff. If users need additional or newer packages they can create a virtualenv and install those, especially for stuff which is on PyPi. Counter-examples are packages which usually are used together, e.g. the tensorboard-profiling plugin in the TF EC. But I don't see that applying here, so I'd take the puristic PoV.